src/overlays/actors/ovl_En_Bee/z_en_bee.c

```diff
@@ -118,121 +118,124 @@ void EnBee_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80B5A854(EnBee* this) {
+void EnBee_SetupIdle(EnBee* this) {
     s32 pad;
-    Vec3f sp48;
-    s16 sp46;
+    Vec3f tmpPos;
+    s16 yawOffset;
 
-    Animation_Change(&this->skelAnime, &object_bee_Anim_00005C, 1.0f, 0.0f,
-                     Animation_GetLastFrame(&object_bee_Anim_00005C), 0, -10.0f);
-    Math_Vec3f_Copy(&sp48, &this->actor.home.pos);
+    Animation_Change(&this->skelAnime, &gBeeFlyingAnim, 1.0f, 0.0f, Animation_GetLastFrame(&gBeeFlyingAnim), 0, -10.0f);
+    Math_Vec3f_Copy(&tmpPos, &this->actor.home.pos);
 
-    sp46 = (this->unk_218 * 0x700) + 0x2000;
-    sp48.x += Math_SinS(sp46) * 50.0f;
-    sp48.y = Rand_ZeroFloat(50.0f) + (this->actor.floorHeight + 30.0f);
-    sp48.z += Math_CosS(sp46) * 50.0f;
+    yawOffset = (this->instanceId * 0x700) + 0x2000;
+    tmpPos.x += Math_SinS(yawOffset) * 50.0f;
+    tmpPos.y = Rand_ZeroFloat(50.0f) + (this->actor.floorHeight + 30.0f);
+    tmpPos.z += Math_CosS(yawOffset) * 50.0f;
 
-    Math_Vec3f_Copy(&this->unk_21C[0], &sp48);
-    Math_Vec3f_Copy(&sp48, &this->actor.home.pos);
+    Math_Vec3f_Copy(&this->moveToDestinations[0], &tmpPos);
+    Math_Vec3f_Copy(&tmpPos, &this->actor.home.pos);
 
-    sp48.x += Math_SinS(sp46 - 0x4000) * 50.0f;
-    sp48.y = Rand_ZeroFloat(50.0f) + (this->actor.floorHeight + 30.0f);
-    sp48.z += Math_CosS(sp46 - 0x4000) * 50.0f;
+    tmpPos.x += Math_SinS(yawOffset - 0x4000) * 50.0f;
+    tmpPos.y = Rand_ZeroFloat(50.0f) + (this->actor.floorHeight + 30.0f);
+    tmpPos.z += Math_CosS(yawOffset - 0x4000) * 50.0f;
 
-    Math_Vec3f_Copy(&this->unk_21C[1], &sp48);
+    Math_Vec3f_Copy(&this->moveToDestinations[1], &tmpPos);
 
-    this->unk_204 = Rand_S16Offset(0x14, 0x1E);
-    this->unk_208 = 0;
-    this->actionFunc = func_80B5A9E8;
+    this->attackDelayTimer = Rand_S16Offset(0x14, 0x1E);
+    this->isHostile = 0;
+    this->actionFunc = EnBee_Idle;
 }
 
-void func_80B5A9E8(EnBee* this, PlayState* play) {
-    Vec3f sp3C;
+void EnBee_Idle(EnBee* this, PlayState* play) {
+    Vec3f nextDestination;
     s32 pad[2];
 
     if ((this->actor.category != ACTORCAT_ENEMY) && (ActorCutscene_GetCurrentIndex() == -1)) {
         func_800BC154(play, &play->actorCtx, &this->actor, ACTORCAT_ENEMY);
     }
 
-    Math_Vec3f_Copy(&sp3C, &this->unk_21C[this->unk_214]);
-    sp3C.x += Math_SinS(this->unk_20C) * 30.0f;
-    sp3C.z += Math_CosS(this->unk_20C) * 30.0f;
+    Math_Vec3f_Copy(&nextDestination, &this->moveToDestinations[this->destinationIndex]);
+    nextDestination.x += Math_SinS(this->targetYaw) * 30.0f;
+    nextDestination.z += Math_CosS(this->targetYaw) * 30.0f;
 
-    if (!(this->unk_218 & 1)) {
-        this->unk_20C += (s16)((s32)randPlusMinusPoint5Scaled(1000.0f) + 4000);
+    if (!(this->instanceId & 1)) {
```

Comment:
Wow this is used weirdly. I suppose you ought to write it as `% 2 == 0` (should match based on similar code, rule is powers of 2 in ifs generally).